WalletWasabi.Tests/UnitTests/StartWasabiOnBootUpTests.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Runtime.InteropServices;
+using WalletWasabi.Fluent.Helpers;
+using Xunit;
+
+namespace WalletWasabi.Tests.UnitTests
+{
+	public class StartWasabiOnBootUpTests
```

Comment:
Everywhere else we used `OnSystemStartup` so I would rename this class to `StartWasabiOnSystemStartupTests`

Reply:
Done.